contents/overlay/drawers/form-drawer/metadata.json

```
@@ -0,0 +1,17 @@
+{
+  "en": {
+    "title": "Form Drawer",
+    "description": "Contact form made by Drawer"
+  },
+  "ja": {
+    "title": "ドロワー製フォーム",
+    "description": "ドロワー製のフォーム"
+  },
```

Here is a suggestion on the title and description.

```suggestion
  "en": {
    "title": "Contact Form Drawer",
    "description": "A drawer component containing a contact form."
  },
  "ja": {
    "title": "お問い合わせフォームのドロワー",
    "description": "お問い合わせフォームを含むドロワーコンポーネント。"
  },
```